### PR TITLE
Fix deneb builder bid HTR

### PR DESCRIPTION
**Pull request by terencechain**
Fixed a few things
- `BuilderBidDeneb` ordering was off. [Reference](https://github.com/ethereum/builder-specs/blob/main/specs/deneb/builder.md#builderbid)
- `BlindedBlobsBundle` 's size is a list and `max_blob_commitments`

### CODE REVIEW

File: proto/prysm/v1alpha1/beacon_block.proto [hunk 1]
```diff
@@ -767,9 +767,9 @@ message SignedBuilderBidCapella {
 
 message BuilderBidDeneb {
     ethereum.engine.v1.ExecutionPayloadHeaderDeneb header = 1;
-    bytes value = 2 [(ethereum.eth.ext.ssz_size) = "32"];
```
Comment (james-prysm) [inline proto/prysm/v1alpha1/beacon_block.proto:770]: with this change I think client/builder/bid.go needs to be updated

Comment (terencechain) [reply proto/prysm/v1alpha1/beacon_block.proto:770]: What change?

Comment (james-prysm) [reply proto/prysm/v1alpha1/beacon_block.proto:770]: I think this code
```
// Value --
func (b builderBidDeneb) Value() []byte {
	return b.p.Value
}

```

Comment (terencechain) [reply proto/prysm/v1alpha1/beacon_block.proto:770]: Oh it's not removed, we just changed the ordering from 2 to 3

Comment (james-prysm) [reply proto/prysm/v1alpha1/beacon_block.proto:770]: oops i totally misread sorry

**Review by james-prysm — APPROVED** [review]

**Pull request merged**